src/ProjectTemplates/Web.ProjectTemplates/content/Worker-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Common",
+    "Worker"
+  ],
+  "name": "Worker Service",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "An empty project template for creating a worker service. This template does not have any content in it.",
```

Comment:
I don't think we need to say that 'This template does not have any content in it.' Unless you had a reason for it?

Reply:
It was copied from Empty. This is the worker equivalent of Empty. Keep or kill?